frontend/src/api/MetricsRepository.js

```diff
@@ -1,9 +1,51 @@
 import { umsRequest } from "./Repository";
+import Vue from 'vue'
 
 const resource = "/metrics";
 
 export default {
     get(queryParams) {
         return umsRequest().then(axiosInstance => axiosInstance.get(`${resource}/${queryParams}`));
-    }
-}
+    },
+    async getTotalActiveUserCount(format) {
+        let yearOfActiveUserCount = Vue.prototype.UserCountCache;
+        if (Object.keys(yearOfActiveUserCount).length === 0) {
```

Comment:
`Object.keys` method throws error when `yearOfActiveUserCount ` is undefined - charts won't load.
I changed it locally to `if (yearOfActiveUserCount === undefined || Object.keys(yearOfActiveUserCount).length === 0)` to make the charts load, but there's probably cleaner way to do it. (I didn't dig into the length check)

Reply:
I think It's because I forgot to commit `Vue.prototype.UserCountCache = {};` in main.js ... gonna do that right now and check if the bug is still present